README.md

```diff
@@ -20,6 +20,16 @@ Strapi is a free and open-source headless CMS delivering your content anywhere y
 - **Database agnostic**. You can choose the database you prefer. Strapi works with SQL databases: PostgreSQL, MySQL, MariaDB, and SQLite.
 - **Customizable**. You can quickly build your logic by fully customizing APIs, routes, or plugins to fit your needs perfectly.
 
+## Documentation websites
+
+The main official Strapi documentation website is hosted at [docs.strapi.io](https://docs.strapi.io).
+
+Additional official Strapi documentation websites cover different content:
+
+- Documentation for the older, unsupported version 3 of Strapi is hosted at [docs-v3.strapi.io](https://docs-v3.strapi.io).
+- Upcoming and/or experimental documentation is hosted at [docs-next.strapi.io](https://docs-next.strapi.io). It will soon start including in-progress documentation for Strapi v5.
```

Comment:
I would suggest replacing "soon" with a vague estimation date like "starting autumn/winter 2023" or something like that. Just so that no matter when users read it, it's still applicable, people know what you're talking about and you don't have to constantly update the README.

Reply:
Great, thank you! I fixed it :)